src/components/Search/index.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { withRouter } from "react-router-dom";
+
+class Search extends React.Component {
+  state = {
+    value: null,
+  };
+  handleChange = (e) => {
+    const value = e.target.value;
+    this.setState({ value: value });
```

Comment:
`const { value } = e.target;`
`this.setState({ value });`

Reply:
Thank you...I will modify it.